syncd/FlexCounter.cpp

```diff
@@ -1454,7 +1454,7 @@ void FlexCounter::runPlugins(
 
     const std::vector<std::string> argv =
     {
-        std::to_string(COUNTERS_DB), // TODO
+        std::to_string(db.getDbId()),
```

Comment:
it db is always counter db, can you rename it to cdb?

Reply:
sure. updated